macOS/DuckDuckGo/Application/AboutPanel.swift

```diff
@@ -109,9 +109,17 @@ final class AboutPanelController {
         panel.contentView = hosting.view
     }
 
-    func show() {
-        panel.makeKeyAndOrderFront(nil)
-        NSApp.activate(ignoringOtherApps: true)
+    private func showPanel() {
+        panel.show()
+    }
+
+    static func show(internalUserDecider: InternalUserDecider) {
```

Comment:
I'd consider marking this method `@MainActor` so that the window lookup handles concurrency well.

Reply:
Sure, I can mark the entire controller as such.